test/xpu_api/random/device_tests/distribution_device_test.pass.cpp

```diff
@@ -0,0 +1,882 @@
+// -*- C++ -*-

```

Comment:
do we want to split this file to correspond to certain distribution?

Reply:
Yes, it'll be better. Done